abcmeta/__init__.py

```diff
@@ -181,7 +183,10 @@ def __init_subclass__(cls):
                 diff_details = _compare_signatures_details(
                     obj_method_signature, derived_method_signature
                 )
-                raise AttributeError(
+                errors.append(
                     "Signature of the derived method is not the same as parent"
                     " class:\r\n{}".format(_prepare_text_to_raise(diff, diff_details))
                 )
+
+        if errors:
+            raise AttributeError("\n\n".join(errors))
```

Comment:
Feel free to suggest a new join string:

```suggestion
            raise AttributeError("\n\n\n".join(errors))
```